scripts/sign_artifacts.sh

```diff
@@ -72,13 +77,17 @@ case $MONGO_OS in
 
   "windows-64")
     setup_garasign_authentication
-    authenticode_sign "release.msi"
-    pgp_sign "release.zip" "release.zip.sig"
+    msifile=$(ls "$glob_base.msi")
+    authenticode_sign "$msifile"
+    zipfile=$(ls "$glob_base.zip")
+    pgp_sign "$zipfile" "$zipfile.sig"
     ;;
 
   *)
+    ls -l # XXX - debug
```

Comment:
```suggestion
```

Reply:
This PR is not ready for review. I didn't request a review on it, did I?